src/index.prod.js

```diff
@@ -9,6 +9,10 @@ import './deprecated';
 // expose global variables
 import './globals';
 
+import VueI18n from "vue-i18n";
+
+Vue.use(VueI18n);
+
```

Comment:
Move this `Vue.use` call after all the file imports, preferably near all the other `Vue.use` statemens further down in this page